python_package/madflow/op_generation.py

```diff
@@ -0,0 +1,335 @@
+from madflow.op_aux_functions import *
+
+n_events = argument("nevents", "const int", 0, False, [])
+forLoopString = "for (int it = 0; it < " + n_events.name + "; it += 1) {"
```

Comment:
Why are these two lines separated (and more importantly executed) at import time?

Reply:
The number of events is not explicitly defined in TensorFlow, as it's implicitly obtained from tensor shapes. As it is used in multiple functions I thought it was better (and less error-prone) to separate these two lines.